CoreFoundation/Base.subproj/DarwinSymbolAliases

```diff
@@ -1,7 +1,7 @@
 
 
 # This is specific to Swift open source; Foundation's NSCFConstantString is used as our constant string class reference
-__TMC15SwiftFoundation19_NSCFConstantString ___CFConstantStringClassReference
+_$s15SwiftFoundation19_NSCFConstantStringCN ___CFConstantStringClassReference
```

Comment:
Just a note here to double check this vs recent discussions about the mangling changing.

Reply:
Yup; it's using `$s`, which is the new prefix. (The other class names in the source have been updated accordingly as well.)